sympy/functions/special/zeta_functions.py

```diff
@@ -463,6 +463,17 @@ def _eval_rewrite_as_dirichlet_eta(self, s, a=1):
     def _eval_rewrite_as_lerchphi(self, s, a=1):
         return lerchphi(1, s, a)
 
+    def _eval_is_finite(self):
+        from sympy.solvers import solve
+        from sympy import Eq
+        if self.args[0].is_real != None:
+            if self.args[0] == 1:
+                return False
+            elif solve(Eq(self.args[0], 1)):
```

Comment:
@asmeurer 
```
>>> b = Symbol('b', positive=True)
>>>zeta(b).is_finite 
None
``` 
because it's domain contain ```b = 1``` also.
now take 
```
```
>>> b = Symbol('b', positive=True)
>>>zeta(b+1).is_finite 
True
``` 
because ```(b+1)>1``` so it's finite
overall, we can put any ```args```  and check it's possible solution is ```1``` then it's output is ```None``` otherwise output is ```True```